src/icon-button.css.ts

```diff
@@ -345,6 +345,26 @@ export const buttonContainer = styleVariants({
     ],
 });
 
+export const interactiveAreaBleed = styleVariants({
+    default: {
+        // we add "!important" to override the margin reset applied by Touchable
```

Comment:
ufff

Reply:
Yeah, it's not awesome :(

Reply:
Discussed offline. Removed the !important and added a prop to BaseTouchable in order to be able to avoid the margin reset if needed